.github/workflows/build.yml

```diff
@@ -40,11 +40,3 @@ jobs:
               set -e
               echo "No changes since last run"
           fi
```

Comment:
Shouldn't there be a git push here?

Reply:
So, for example, if I built it locally and tested before committing, then it would not push here since there will not be any changes

Reply:
🤦 lol sorry.